revolv/project/stats.py

```diff
@@ -0,0 +1,124 @@
+"""
+Module for encapsulating functions related to project statistics (aka
+computing the effects of projects based on their stated kilowatt output).
+"""
+from django.db.models import Sum
+
+
+class KilowattStatsAggregatorException(Exception):
+    pass
+
+
+class KilowattStatsAggregator(object):
+    """
```

Comment:
They're kilowatts, but I agree that we should make sure kilowatt/kWh in docstrings. Will do that!